vite.config.ts

```diff
@@ -6,4 +9,17 @@ export default defineConfig({
     open: true,
   },
   plugins: [react()],
+  test: {
+    environment: 'jsdom',
+    setupFiles: ['./src/setupTest.ts'],
```

Comment:
```suggestion
    setupFiles: './src/setupTest.ts',
```
It alse can ba a single string